docs/developer-guide/build-service.md

```diff
@@ -84,53 +84,13 @@ Repeat this process for the footer:
 
 The build service is capable of including more than one component in the same bundle, so you can simply add multiple modules into the same URL.  Here's an example:
 
-	<link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-header@^3.0.0,o-footer@^3.0.0" />
+	<link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-fonts@^1,o-ft-icons@^2,o-header@^3.0.0,o-footer@^3.0.0" />
 
 It's important that you do this, so that any CSS that is shared between the header and footer (there's quite a bit) isn't downloaded twice.
 
 <aside>Remember that when you change the modules you are requesting in your build service tags, it may take a few minutes to build the resulting bundle of code.  Be patient and hit 'Run with JS' a few times until the styling appears.</aside>
 
 
-### Loading fonts and icons
-
-Edit the `<link>` tag to load styles for these modules (you'll need to look them up in the registry as you did for header and footer):
-
-* `o-ft-icons`
-* `o-fonts`
-
-You should have something like this:
-
-	<link rel="stylesheet"
-	      href="//build.origami.ft.com/bundles/css?modules=o-header@^3.0.0,o-footer@^3.0.0,o-fonts@^1.4.0,o-ft-icons@^2.1.1" />
```

Comment:
Worth adding a comment into the boilerplate code certainly, though you are not actually loading o-fonts by default, you're just applying them if they have been loaded.  Perhaps you should add o-fonts in the build service call in the boilerplate, and comment that.  But we should still mention it in the tutorial, pointing out both the build service URL and the snippet of inline CSS.

I wouldn't worry too much about catering for people who don't read the instructions.  There's only so much you can do, this is a tutorial, after all.  There are certainly cases where we should not expect people to RTFM in order to know something - notably if we can just tell them it at exactly the point it becomes relevant, like in an error message from obt.  But if they have opted to read and follow a tutorial, and then don't, well that's to a large extent their problem!  We can obviously try and make the tutorial easier to follow but there's always a risk that you'll actually make it less useful to people who are following it comprehensively.